File: S3/Programmation/Python/voiture/Model/Ville.py
class Ville :
    def __init__(self,id,nom):
        self._id=id
        self._nom=nom
     
    @property
    def id(self):
        return self._id
    
    @id.setter
    def id(self,id):
        self._id=id
    
    @property
    def nom(self):
        return self._nom
    
    @id.setter
    def nom(self,nom):
        self._nom=nom
     
    def __str__(self):
        return f"ville (id={self.id}, nom={self.nom})"

File: S3/Programmation/Post-Gis/src/model/Ville.java
package model;

public class Ville {
    private int id;
    private String nom;
    private double latitude;
    private double longitude;

    public Ville(int id, String nom, double latitude, double longitude) {
        this.id = id;
        this.nom = nom;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public int getId() { return id; }
    public String getNom() { return nom; }
    public double getLatitude() { return latitude; }
    public double getLongitude() { return longitude; }
}
